src/apiKey.js

```diff
@@ -1,5 +1,7 @@
 var API_KEY = 'API_KEY'
 var API_KEY_LOG_TYPE = 'ApiKeyLog'
+var ADD_API_KEY_ACTION = 'AddApiKey'
+var VALIDATE_API_KEY_ACTION = 'ValidateApiKey'
```

Comment:
should there be `REMOVE_API_KEY_ACTION` also?

Reply:
Good point, added logging when a user removes an API key.